app/subscriber/src/features/home/Home.tsx

```diff
@@ -57,11 +57,14 @@ export const Home: React.FC = () => {
   const fetchResults = React.useCallback(
     async (filter: MsearchMultisearchBody) => {
       try {
+        if (sortBy === 'source') {
+          filter.sort = [{ 'source.sortOrder': 'asc' }, { publishedOn: 'desc' }];
+        }
         const res: any = await findContentWithElasticsearch(filter, false);
         setContent(res.hits.hits.map((h: { _source: IContentModel }) => h._source));
       } catch {}
     },
-    [findContentWithElasticsearch],
+    [findContentWithElasticsearch, sortBy],
```

Comment:
I don't think we want to send the `source.sortOrder` value to Elasticsearch.  It will skew the search results importance.  Can you instead just sort the results of the search by the `source.sortOrder`?

Reply:
I don't like the idea of port processing information on the frontend, if we can get it already processed at the backend.
This update orders the result, but I would still prefer to have it already ordered by a DB engine then a javascript sort function.